WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -166,6 +176,12 @@ public class ReaderScreen: ScreenObject {
         return self
     }
 
+    public func openLikes() -> Self {
```

Comment:
WDYT of adding `Tab` into the name?

```suggestion
    public func openLikesTab() -> Self {
```

Reply:
the reason i didn't was there were a few existing ones (e.g. `openDiscover`) that didn't use `tab` in the name but yeah updating them would make them clearer, i'll do that in a separate PR (and update all the tabs on the screen) along with the naming suggestion below (i'll move the "clean up"/name changes into a new PR as they will cause changes in screen and test files)